frontend/src/surveys/components/SurveysList/SurveysList.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Survey from '../Survey/Survey.jsx';
+
+const SurveysList = ({ surveys, onSurveyClick }) => (
+    <div className='row surveys-list'>
+        {surveys.map(survey =>
+            (<Survey
+                key={survey.id}
+                {...survey}
+                onClick={() => onSurveyClick()}
```

Comment:
Avoid using arrow functions in render method.
https://medium.freecodecamp.org/why-arrow-functions-and-bind-in-reacts-render-are-problematic-f1c08b060e36

How you determine what survey is clicked?